sound/soc/codecs/max98363.c

```diff
@@ -176,6 +176,8 @@ static int max98363_io_init(struct sdw_slave *slave)
 	ret = regmap_read(max98363->regmap, MAX98363_R21FF_REV_ID, &reg);
 	if (!ret) {
 		dev_info(dev, "Revision ID: %X\n", reg);
+	} else {
```

Comment:
I also don't understand what happens if the regmap_read() returns an error, we keep going?

What is the purpose of this check, something does not add up or I need more coffee.

Reply:
> need pm_runtime_mark_last_busy(dev); here
> 
> or better, add a label and jump to the end of the function.

Sure, I will add a lebel.

> I also don't understand what happens if the regmap_read() returns an error, we keep going?
> What is the purpose of this check, something does not add up or I need more coffee.

Originally, it will keep going only when regmap_read() returns an error, and that was a bug. That is the reason why max98363 keep active.